com.unity.render-pipelines.universal/Runtime/Passes/PostProcessPass.cs

```diff
@@ -1391,6 +1387,70 @@ void RenderFinalPass(CommandBuffer cmd, ref RenderingData renderingData)
 
             var colorLoadAction = cameraData.isDefaultViewport ? RenderBufferLoadAction.DontCare : RenderBufferLoadAction.Load;
 
+            bool isFxaaEnabled = (cameraData.antialiasing == AntialiasingMode.FastApproximateAntialiasing);
+            bool isUpscaleSetupTextureUsed = false;
+
+            if (cameraData.imageScaling != ImageScaling.None)
+            {
+                // Make sure to remove any MSAA and attached depth buffers from the temporary render targets
+                var tempRtDesc = cameraData.cameraTargetDescriptor;
+                tempRtDesc.msaaSamples = 1;
+                tempRtDesc.depthBufferBits = 0;
+
+                m_Materials.upscaleSetup.shaderKeywords = null;
+
+                // When FXAA is enabled in scaled renders, we execute it in a separate blit since it's not designed to be used in
+                // situations where the input and output resolutions do not match.
+                if (isFxaaEnabled)
+                {
+                    m_Materials.upscaleSetup.EnableKeyword(ShaderKeywordStrings.Fxaa);
+
+                    isUpscaleSetupTextureUsed = true;
+
+                    cmd.GetTemporaryRT(ShaderConstants._UpscaleSetupTexture, tempRtDesc, FilterMode.Point);
```

Comment:
when merging to master this (and any other RT management code) needs to be changed to use RTHandles as we have deprecated GetTemporaryRT

cc @sandy-carter-unity

Reply:
@manuele-bonanno & @sandy-carter-unity do my RTHandle changes in 4900e6e look ok?